content/docs/0.6.0/installation/k8s-support/index.md

```diff
@@ -21,7 +21,11 @@ Keptn versions are expressed as `x.y.z`, where `x` is the major version, `y` is
 | Keptn Version /<br>Installation       | Kubernetes      | AKS             | EKS         | GKE             | OpenShift | Minikube              | MicroK8s<br>(experimental) | Minishift<br>(experimental) |
 |---------------------------------------|-----------------|-----------------|-------------|-----------------|-----------|-----------------------|----------------------------|-----------------------------|
 | 0.6.x / <br>Control & Execution Plane | 1.15.x - 1.13.x | 1.15.x          | 1.15 - 1.14 | 1.15.x - 1.14.x | 3.11      | -                     | -                          | 1.34.2<br>(K8s: 1.11.x)     |
-| 0.6.x / <br>Control Plane             | 1.16.x - 1.13.x | 1.16.x - 1.15.x | 1.15 - 1.14 | 1.15.x - 1.14.x | 3.11      | 1.2 <br>(K8s:1.15.x) | 1.18.x                     | 1.34.2<br>(K8s: 1.11.x)     |
+| 0.6.x / <br>Control Plane             | 1.16.x - 1.13.x | 1.16.x - 1.15.x | 1.15 - 1.14 (1) | 1.15.x - 1.14.x (1) | 3.11      | 1.2 <br>(K8s:1.15.x) | 1.18.x                     | 1.34.2<br>(K8s: 1.11.x)     |
+
+**Remarks:**
+
+* (1) EKS and GKE do not provide K8s 1.16 clusters (checked on: 9th April, 2020)
```

Comment:
GKE does.
![Screenshot from 2020-04-09 16-48-35](https://user-images.githubusercontent.com/56065213/78908146-fb144180-7a81-11ea-8992-a79f2211f912.png)

Reply:
Ohh right. I never selected **Release channel** - there 1.16 is hidden. 
I will try it out.